content-src/components/TimelinePage/TimelineBookmarks.js

```diff
@@ -3,13 +3,25 @@ const {connect} = require("react-redux");
 const {selectSpotlight} = require("selectors/selectors");
 const GroupedActivityFeed = require("components/ActivityFeed/ActivityFeed");
 const Spotlight = require("components/Spotlight/Spotlight");
+const {RequestMoreBookmarks} = require("actions/action-manager").actions;
+const LoadMore = require("components/LoadMore/LoadMore");
 
 const TimelineBookmarks = React.createClass({
+  getMore() {
+    const bookmarks = this.props.Bookmarks.rows;
+    if (!bookmarks.length) {
+      return;
+    }
+    const beforeDate = bookmarks[bookmarks.length - 1].lastModified;
+    this.props.dispatch(RequestMoreBookmarks(beforeDate));
```

Comment:
This dispatches a `RECENT_BOOKMARKS_REQUEST` with the beforeDate as a query option